tests/foreman/api/test_subscription.py

```diff
@@ -240,6 +243,53 @@ def test_positive_subscription_status_disabled(self):
         host_content = entities.Host(id=host_id).read_raw().content
         assert 'Simple Content Access' in str(host_content)
 
+    @pytest.mark.tier2
+    @pytest.mark.usefixtures('golden_ticket_host_setup')
+    @pytest.mark.usefixtures('rhel77_contenthost_class')
+    def test_sca_end_to_end(self):
+        """Perform end to end testing for Simple Content Access Mode
+
+        :id: c6c4b68c-a506-46c9-bd1d-22e4c1926ef8
+
+        :expectedresults: All end to end tests pass and clients have access
+        to repos without needing to add subscriptions
+
+        :CaseImportance: Medium
+        """
+        self.content_host.install_katello_ca()
+        self.content_host.register_contenthost(self.org_setup.label, self.ak_setup.name)
+        assert self.content_host.subscribed
+        # Check to see if Organization is in SCA Mode
+        assert entities.Organization(id=self.org_setup.id).read().simple_content_access is True
```

Comment:
I ran this locally and for some reason I can't get passed this, but I modified it with `entities.Organization(id=self.org_setup.id).read_json()['simple_content_access']`.  It be my local setup being incorrect, but I'll let you decide which to use.

Reply:
Fails for me on `.read()`.  What's the result when you're running the stand-alone job?

Reply:
you can trigger via the prt